spec/features/navigation_spec.rb

```diff
@@ -129,5 +129,18 @@
         expect(page).to_not have_content("Cart")
       end
     end
+
+    it "I see the same links as a regular user and a link to merchant dashboard" do
+      fill_in :email, with: @merchant.email
+      fill_in  :password, with: @merchant.password
+      click_button "Login"
+
+      within 'nav' do
+        expect(page).to have_content("Logged in as #{@merchant.name}")
+        expect(page).to have_link("Profile")
+        expect(page).to have_link("Logout")
+        expect(page).to have_link("Merchant Dashboard")
+      end
```

Comment:
Maybe we should also add expectation that they have link to Cart as well since I believe merchants need access to that too?